README.md

```diff
@@ -50,7 +50,7 @@ Once you have Carthage [installed](#installing-carthage), you can begin adding f
 1. Run `carthage update`. This will fetch dependencies into a [Carthage/Checkouts][] folder and build each one.
 1. On your application targets’ “General” settings tab, in the “Embedded Binaries” section, drag and drop each framework you want to use from the [Carthage/Build][] folder on disk.
 
-##### If you're building for iOS
+##### If you're building for iOS (for tvOS, read `iOS` as `tvOS`)
```

Comment:
Right, good point. This also applies for `watchOS`. Can you maybe change the wording to something like "if you're building for iOS, tvOS, or watchOS..."